src/containers/BooksForm.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import uuidv4 from 'uuid/v4';
+import PropTypes from 'prop-types';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
```

Comment:
i think there should be a default category to this state instead of an empty string

Reply:
Anyways, you can probably handle it another way 😊